assets/sass/05-blocks/search/_editor.scss

```diff
@@ -33,6 +46,75 @@
 			background-color: var(--button--color-background);
 			color: var(--button--color-text);
 		}
+
+		&.has-icon {
+			padding: calc(2 * var(--button--padding-vertical)) calc(0.5 * var(--button--padding-horizontal));
+
+			svg {
+				width: 40px;
+				height: 40px;
+			}
+		}
+
+		.has-background:not(.has-background-background-color) &,
+		[class*="background-color"]:not(.has-background-background-color) &,
+		[style*="background-color"]:not(.has-background-background-color) &,
+		.wp-block-cover[style*="background-image"] & {
+			color: var(--global--color-white);
```

Comment:
Shouldn't the color here be changed only for darker backgrounds? I think this color rule should apply to the same selectors from L31-33 above: 

```
.has-background.has-gray-background-color &,
.has-background.has-dark-gray-background-color &,
.has-background.has-black-background-color &
```